src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -215,6 +215,13 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
     @Override public Profiler getProfiler() {
         return this.theProfiler;
     }
+    
+    //Overwriting original World.markChunkDirty()
+    public void markChunkDirty(BlockPos pos, TileEntity unusedTileEntity) {
```

Comment:
[Are you sure?](https://github.com/Barteks2x/CubicChunks/blob/7e27ce672ef89b9c529d5eca5f6d5fc5e8b65fec/src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld_HeightLimits.java#L79)